_projects/ashlar.md

```diff
@@ -0,0 +1,42 @@
+---
+published: false # change to true once ready to post
+
+title: "Ashlar" # project title inside quotes
+excerpt: "Ashlar schema editor in Django admin" # shows on project list page
+seo_description: "" # goes in project meta description
+
+# project attributes
+requirements: # bullet list of requirements – one requirement per line – follow below format
+  - "Python experience"
+  - "Django experience preferred"
+  - "Javascript experience preferred"
+tags: # one tag per line – spaces are allowed in tags – follow below format
+  - "python"
+  - "django"
+  - "jsonb"
+  - "postgresql"
+difficulty: "hard" # easy, medium, hard – pick one
+mentors: # github username without the @ – example: designmatty
+  - "ddohler"
+  - ???
+
+# This file uses Kramdown. See https://kramdown.gettalong.org/syntax.html for syntax
+---
+
+# MILESTONES
+
+## PREPARATION
+
+- Familiarization with Ashlar project
+- 
+
+## CODING PHASE 1
+
+- Bring Ashlar project dependencies up to date
+- Support Python 3 (?)
+- ??? more detail
```

Comment:
The mention of jsonb reminded me: the project uses https://github.com/azavea/djsonb because it predated native Django support for jsonb.  It's been a while, but my memory is that the way we implemented searching/filtering had some unfortunate limitations and was moderately torturous to change or maintain.

So I think refactoring to use Django's built in jsonb functionality would be worthwhile (and if it's gong to happen it should happen up front, to avoid the aforementioned maintenance cost dragging on other tasks).